diff --git a/node_modules/openid-client/lib/client.js b/node_modules/openid-client/lib/client.js
index e7a5716..ed4df7e 100644
--- a/node_modules/openid-client/lib/client.js
+++ b/node_modules/openid-client/lib/client.js
@@ -701,6 +701,8 @@ module.exports = (issuer, aadIssValidation = false) => class Client extends Base
 
     idToken = String(idToken);
 
     const timestamp = now();
     const { protected: header, payload, key } = await this.validateJWT(idToken, expectedAlg);
 
@@ -734,12 +736,15 @@ module.exports = (issuer, aadIssValidation = false) => class Client extends Base
     }
 
     if (returnedBy === 'authorization') {
-      if (!payload.at_hash && tokenSet.access_token) {
-        throw new RPError({
-          message: 'missing required property at_hash',
-          jwt: idToken,
-        });
-      }
+      // @see https://community.auth0.com/t/at-hash-not-added-to-id-token-after-successful-authorization/8953/2
+      // getting Auth0 and Cypress to seems impossible with these lines
+      // 
+      // if (!payload.at_hash && tokenSet.access_token) {
+      //   throw new RPError({
+      //     message: 'missing required property at_hash',
+      //     jwt: idToken,
+      //   });
+      // }
 
       if (!payload.c_hash && tokenSet.code) {
         throw new RPError({
